source/appModules/explorer.py

```diff
@@ -251,15 +251,17 @@ def _get_TextInfo(cls):
 
 class WorkerW(IAccessible):
 	def event_gainFocus(self):
+		if ((winVersion.winVersion.major, winVersion.winVersion.minor) != (6, 1)):
+			return

```

Comment:
It would be helpful to describe the problem with handling Win 8 here.

Reply:
I've updated the commends however rather than trying to explain the Win 8 problem which I frankly do not understand and therefore can't explain clearly I've concentrated on the fact that on anything else than Windows 7 the event is irrelevant and can be ignored